MOSH_README.md

```diff
@@ -0,0 +1,64 @@
+# Using `mosh` with Fluent Terminal
+
+`mosh` can be used in Fluent Terminal in two ways:
+
+ 1. Through GUI, by turning ***Use mosh*** option in *SSH Connection* dialog;
+ 2. As a custom command, in *Quick Launch* dialog.
+
+The first way, through GUI is very simple and self-descriptive, and it will not be explained here. Using `mosh` as a custom command is more complicated, but also offers much more options. The simplest `mosh` command you can enter is:
+
+    mosh user@host
+
+This will initiate a `mosh` session with default options (i.e. port range will default to `60000:61000`, etc.).
+
+`mosh` additionally supports many more command-line arguments. In fact, it's almost completely compliant to [mosh(1) - Linux man page](https://linux.die.net/man/1/mosh). `mosh` is defined there as:
```

Comment:
`mosh` supports a variety of command-line arguments as follows:

Reply:
Not sure that I understand. You want me to remove part which explains what isn't supported?